tools/test/travis-ci/doxy-spellchecker/ignore.en.pws

```diff
@@ -118,3 +118,5 @@ Vin
 Vref
 ssid
 _doxy_
+instantiation
+KVStore
```

Comment:
```suggestion
instantiation
KVStore
_doxy_
```

Reply:
After doxy whatever listed is not considered by Doxygen spell checker

Reply:
Ah thanks, I thought that was a word too 😛